### PR TITLE
test(ui-utils): add tests for validation utils & refactor imports

### PR DESCRIPTION
**Pull request by hdinia**
Added tests for validation utility functions

- String validation tests
- Number validation tests
- Array validation tests

- Split validation utils into separate files for better organization
- Updated Vite config to support relative imports
- Added '@' alias for the src directory

- Updated `tsconfig.json` to support the new import alias
- Refactored import statements across the project to use the new '@' alias
- Updated dependencies if necessary

### CODE REVIEW

File: webapp/src/utils/validation/string.ts [hunk 1]
```diff
@@ -13,13 +13,6 @@
  */
 
 import { t } from "i18next";
-import * as R from "ramda";
-
-type ValidationResult = string | true;
```
Comment (skamril) [inline webapp/src/utils/validation/string.ts:18]: Remove (see other comment)

File: webapp/src/utils/validation/number.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import { t } from "i18next";
+
+type ValidationResponse = string | true;
```
Comment (skamril) [inline webapp/src/utils/validation/number.ts]: ~~Use the `ValidationResult` from types.ts instead (see other comment)~~

Use `ValidateResult` fromn react-hook-form.

File: webapp/tsconfig.json [hunk 3]
```diff
@@ -9,6 +9,10 @@
     ],
     "module": "ESNext",
     "skipLibCheck": true,
+     "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"]
```
Comment (skamril) [inline webapp/tsconfig.json:14]: Not `./src/`? Did you test the build prod?

File: webapp/tsconfig.json [hunk 4]
```diff
@@ -9,6 +9,10 @@
     ],
     "module": "ESNext",
     "skipLibCheck": true,
+     "baseUrl": ".",
```
Comment (skamril) [inline webapp/tsconfig.json:12]: Format the file

**Review by skamril — CHANGES_REQUESTED** [review]

File: webapp/tsconfig.json (hunk 3 above)
Comment (hdinia) [reply webapp/tsconfig.json:14]: build is ok

**Review by skamril — APPROVED** [review]

**Pull request merged**